modes/parkour/interface.lua

```diff
@@ -587,12 +588,19 @@ onEvent("ChatCommand", function(player, msg)
 
 		save_update = os.time() + 60000 * 3 -- 3 minutes
 		translatedChatMessage("action_within_minute", player)
+			
+	elseif cmd == "map" then
+		if not ranks.mapper[player] then return end
```

Comment:
I meant to create a whole new permission, not explicitly checking if the player is a mapper. This way it will be allowed to mappers only. Go and create a new permission in ranks.lua

Reply:
im gonna kill you 😃